code/src/components/CkeckboxToDoList.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable max-len */
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import tasks from 'reducers/tasks';
+
+const ToDoList = () => {
```

Comment:
Do you have two components that are named differently but have the same content?  It looks like bother ToDoList.js and CheckboxToDoList.js have the same content?  Is that right?